Subpatch 4-2-24

rosieposieeee opened the pull request:
<!-- Please read these guidelines before opening your PR: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
<!-- The text between the arrows are comments - they will not be visible on your PR. -->

## About the PR
<!-- What did you change in this PR? -->

- removed lockable crate go fuck yourself. use the COUNTER. **_THE COUNTER!!!!!!_**
- wrenched the med paper (lol)
- added red phone to bridge
- flipped spare ID so it is accessible. haha oops
- goodbye spare bibles
- added bridge fax machine
- gave sec checkpoints two buttons for N/S blast doors
- added atmos into the court defendant/prosecutor boxes (lol)
- one billion more station maps
- gave chem counter a smartfridge and removed the doors
- added a 'barrier stockpile' by singulo
- gave sec jetpacks (honest to god thought they came in the sec suit storage for some reason)
- tweaked salv dock a bit
- removed salv shuttle console which definitely shouldnt still be there lol
- used black shuttle tiles in more places instead of reinf
- removed that one hall bush outside of the court that always annoys me
- i felt like it was a bit unfair that i moved basically all of the oxygen and nitrogen canisters inside so i sprinkled some oxygen tanks around the various solars as a treat for the antags

## Why / Balance
<!-- Why was it changed? Link any discussions or issues here. Please discuss how this would affect game balance. -->

my joker's trick

rosieposieeee committed:
tweaks

- removed lockable crate go fuck yourself. use the COUNTER.
- added red phone to bridge
- flipped spare ID so it is accessible. haha oops
- goodbye bibles
- added bridge fax machine
- gave sec checkpoints two buttons for N/S blast doors
- added atmos into the court defendant/prosecutor boxes (lol)
- one billion more station maps
- gave chem counter a smartfridge

IamVelcroboy commented:
Test fail is valid. Looks like you need to clean the invalids from relinking buttons probably

rosieposieeee commented:
> Test fail is valid. Looks like you need to clean the invalids from relinking buttons probably

pain & evil on planet earth

rosieposieeee committed:
fix invalids

rosieposieeee committed:
Update submarine.yml

- wrenched the med paper (lol)
- added a 'barrier stockpile' by singulo
- gave sec jetpacks (honest to god thought they came in the sec suit storage for some reason)
- tweaked salv dock a bit
- removed salv shuttle console which definitely shouldnt still be there lol
- used black shuttle tiles in more places instead of reinf
- removed that one hall bush outside of the court that always annoys me

IamVelcroboy commented:
Good luck!
